mmv1/third_party/terraform/services/networkservices/resource_network_services_gateway_test.go

```diff
@@ -358,6 +360,7 @@ resource "google_network_services_gateway" "foobar" {
   location                             = "us-east1"
   addresses                            = ["10.128.0.99"]
   type                                 = "SECURE_WEB_GATEWAY"
+  routing_mode                         = "EXPLICIT_ROUTING_MODE"
```

Comment:
Can you please remove this line to test if the default value handing is working with `default_from_api: true` ?

Reply:
Can you please remove this line to see if the test will pass without the new field?

We don't want to add the optional field to all of the test cases.

Reply:
Done